api/redisfailover/v1alpha2/types.go

```diff
@@ -31,6 +31,7 @@ type RedisFailoverSpec struct {
 	// NodeAffinity defines the rules for scheduling the Redis and Sentinel
 	// nodes
 	NodeAffinity *corev1.NodeAffinity `json:"nodeAffinity,omitempty"`
+	Tolerations []corev1.Toleration `json:"tolerations,omitempty"`
```

Comment:
Could you add a comment about this new parameter of the spec?

Reply:
Tolerations provides a way to schedule Pods on Tainted Nodes

Reply:
I meant inside the code, so everyone can read it (like the rest of parameters)

Reply:
Sure, done. Sorry